src/mediarithmics/plugins/audience-feed-connector/AudienceFeedConnectorBasePlugin.ts

```diff
@@ -109,6 +112,10 @@ export abstract class AudienceFeedConnectorBasePlugin extends BasePlugin<Audienc
     instanceContext: AudienceFeedConnectorBaseInstanceContext
   ): Promise<UserSegmentUpdatePluginResponse>;
 
+  protected abstract onBatchUpdate(
+    request: BatchUpdateRequest<unknown>
```

Comment:
May be you should pass instanceContext like others signature above ?

Reply:
Make complete sense